desktop/notification/fdo.go

```diff
@@ -180,10 +228,17 @@ func processNotificationClosed(sig *dbus.Signal, observer Observer) error {
 	if !ok {
 		return fmt.Errorf("expected second body element to be uint32, got %T", sig.Body[1])
 	}
-	return observer.NotificationClosed(ID(id), CloseReason(reason))
+	srv.lastRemove = timeNow()
+	localID, ok := srv.serverToLocalID[id]
```

Comment:
minor: this is the only place where `serverToLocalID` is used. Since it's not in a performance critical path, and given that we will never have many items in the map, I'd rather remove `serverToLocalID` and just iterate the `localToServerID` map here.

Reply:
IMHO I think the code as-is is fine, it's actually less code to store the lookup when we get the server ID back than it is to have a for loop looking for it in the localToServerID map

Reply:
yeah, what Ian said was exactly my line of thinking, so I'd keep it for that reason.